app/controllers/staff_controller.rb

```diff
@@ -1,4 +1,4 @@
-class StaffController < StaffAreaController
```

Comment:
Why this change? If there is some before action in `StaffDashboardController` that needs to be in more places, maybe it should be moved into `StaffAreaController`? Otherwise I fear that if StaffDashboardController will be the accidental implementation for some controller actions instead of causing an error

Reply:
`StaffDashbaordController` has the `before_action :default_space` which sets the space for the staff based on their latest tap. I can move that action to `StaffAreaController`